ui/app/components/variable-form.hbs

```diff
@@ -72,7 +72,7 @@
           Please choose a different path, or
           <LinkTo
             @route="variables.variable.edit"
-            @model={{this.duplicatePathWarning.path}}
+            @model={{concat this.duplicatePathWarning.path "@" (or this.variableNamespace "default")}}
```

Comment:
Side-effect: noticed this had stopped working in namespace-less environments.

Reply:
PS: hey here's that "this variable path already exists" warning I mentioned a comment ago!